spec/features/admin_deletes_a_beer_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.